src/js_native_api.h

```diff
@@ -1,8 +1,9 @@
 #ifndef SRC_JS_NATIVE_API_H_
 #define SRC_JS_NATIVE_API_H_
 
-#include <stddef.h>
-#include <stdbool.h>
+// This file needs to be compatible with C compilers.
+#include <stddef.h>   // NOLINT(modernize-deprecated-headers)
+#include <stdbool.h>  // NOLINT(modernize-deprecated-headers)
```

Comment:
16:37:49 src/js_native_api_types.h:6:  Unknown NOLINT error category: modernize-deprecated-headers  [readability/nolint] [5]
16:37:49 src/js_native_api_types.h:6:  At least two spaces is best between code and comments  [whitespace/comments] [2]

Reply:
@refack is working on cpplint to make it support `clang-tidy` rule :)

Reply:
> @refack is working on cpplint to make it support `clang-tidy` rule :)

It's actually as simple as adding `-readability/nolint` filter

I've also opened a PR to update cpplint, and more our configuration to a config file:
Ref: https://github.com/nodejs/node/pull/27098